sdk/storage/azure-storage-blob/azure/storage/blob/_shared/policies.py

```diff
@@ -241,6 +243,27 @@ def on_response(self, request, response, **kwargs):
                 _LOGGER.debug("Failed to log response: %s", repr(err))
 
 
+class StorageUserAgentPolicy(SansIOHTTPPolicy):
```

Comment:
We should make sure that our core UA policy takes the appropriate parameters. We should only have to specify the library name and version.

Reply:
Yeah - the core UA policy needs updating to reflect the latest guidelines.
For the next preview we should be able to simple use the core UA policy.
I've opened an issue here:
https://github.com/Azure/azure-sdk-for-python/issues/6163